e2e/tests/dump-restore-test.rs

```diff
@@ -288,47 +294,80 @@ fn dump_db(container: &PostgresContainer, db: &str, username: &str, path: &Path)
 /// * `container` - the docker container running postgres
 /// * `db` - the database to connect psql to with the `-d` flag to run the restore against
 /// * `username` - the user to run the restore script
+/// * `path` - the path to the logical dump file on the host
 ///
 fn restore_db(container: &PostgresContainer, db: &str, username: &str, path: &Path) {
     let dump = Path::new("/tmp/dump.sql");
     copy_in(container, path, dump);
-    psql_file(container, db, username, dump);
+    let exit = Command::new("docker")
+        .arg("exec")
+        .arg(container.id())
+        .arg("psql")
+        .args(["-U", username])
+        .args(["-d", db])
+        .arg("--no-password")
+        .arg("--no-psqlrc")
+        .arg("--no-readline")
+        .arg("--echo-all")
+        // do not ON_ERROR_STOP=1. some statements in the restore will fail. for example, setting
+        // the comment on the timescaledb extension. this is highly unfortunate, but seemingly
+        // unavoidable according to googling. eat the errors and continue. we will rely on the
+        // snapshot to determine whether the restore was successful
+        //.args(["-v", "ON_ERROR_STOP=1"]) don't uncomment me!
```

Comment:
we should remember to document this. Can we get a list of expected errors for documentation?

Reply:
yep